### PR TITLE
blog post for cordova-android@7.1.4 patch release

### PR DESCRIPTION
**Pull request by brodybits**
(patch release coming soon)

### CODE REVIEW

File: www/_posts/2018-11-23-cordova-android-7.1.4.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+layout: post
+author:
+    name: Chris Brody
+    url: https://twitter.com/brodybits
+title:  "Cordova Android 7.1.4 Released!"
+categories: announcements
+tags: news releases
+---
+
+We are happy to announce that `Cordova Android 7.1.4` has been released! This release resolves a couple more issues releated to plugin compatibility.
```
Comment (janpio) [inline www/_posts/2018-11-23-cordova-android-7.1.4.md]:
```suggestion
We are happy to announce that `Cordova Android 7.1.4` has been released! This release resolves a couple more issues related to plugin compatibility.
```

**Comment by janpio** [comment]
What do you think about being a bit more honest, like

> This fixes a bug we unfortunately introduced with our 7.1.3 release a few days ago. Thanks for the quick reports on GitHub issues and the help in identifying the problem.

**Comment by brodybits** [comment]
I just pushed changes to make the description more candid (honest), as suggested, with other minor edits. Quick review would be appreciated, apologies for the pressure.

**Comment by janpio** [comment]
TIL "candid"

File: www/_posts/2018-11-23-cordova-android-7.1.4.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+layout: post
+author:
+    name: Chris Brody
+    url: https://twitter.com/brodybits
+title:  "Cordova Android 7.1.4 Released!"
+categories: announcements
+tags: news releases
+---
+
+We are happy to announce that Cordova Android `7.1.4` has been released! This resolves an issue that was introduced in the recent `7.1.3` release. Thanks to the user community for the quick reports on GitHub issues and the help in identifying the problem.
```
Comment (janpio) [inline www/_posts/2018-11-23-cordova-android-7.1.4.md]:
```suggestion
We are happy to announce that Cordova Android `7.1.4` has been released! This resolves a bug that was introduced in the recent `7.1.3` release. Thanks to our users for the quick reports on [GitHub issues](https://github.com/apache/cordova-android/issues) and the help in identifying the problem.
```

**Comment by brodybits** [comment]
I will merge to publish once cordova-android@7.1.4 is approved and published. 1-2 more votes on the cordova-android@7.1.4 VOTE thread would be appreciated.

**Pull request merged**